web/themes/joinup/joinup_theme.theme

```diff
@@ -19,13 +19,77 @@ use Drupal\file\Entity\File;
 use Drupal\image\Entity\ImageStyle;
 
 /**
+ * Check if route is special.
+ *
+ * @param \Drupal\Core\Routing\RouteMatchInterface $route_match
+ *   The route object.
+ *
+ * @return array
+ *   The route type.
+ */
+function joinup_check_route(RouteMatchInterface $route_match) {
+  $route_name = $route_match->getRouteName();
+
+  // Declare available types.
+  $types = [
+    'create' => FALSE,
+    'tabs' => FALSE,
+  ];
+
+  // Check if page has a create form.
+  $create_content_routes = [
+    'collection.propose_form',
+    'rdf_entity.propose_form',
+  ];
+
+  // Check if string has 'add' on the end.
+  if (preg_match('/.add$/', $route_name)) {
```

Comment:
I checked all creation routes:
outside workflow:
* create content form has route "node.add"
* create rdf form has route "rdf_entity.rdf_add"
In workflow, content and rdf add on solution and collection level has similar routes:
ex. joinup_news.rdf_entity_news.add, custom_page.collection_custom_page.add, asset_release.solution_asset_release.add

So all of these routes match the pattern. Only propose collection and propose solution forms don't match this pattern, but they are listed in array $create_content_routes

rdf_entity.propose_form